### DISCUSSION TITLE
Help with Custom Input

### DISCUSSION THREAD

**Discussion by mwlai288** [react-hook-form/react-hook-form#4284] [Q&A] [answered]
I have this custom Text Input and I'm getting this error on the ref prop.

Component is like this
![Screenshot 2021-02-25 140207](https://user-images.githubusercontent.com/26555439/109203441-25422100-7772-11eb-985e-2949eab8e04c.png)

formErrors is an external file where I have the validations for each name.

When hovering over ref, these are the messages I get
![err](https://user-images.githubusercontent.com/26555439/109203541-486cd080-7772-11eb-9e20-558a429661a6.png)

What can I do to fix this? I have no clue what to do

**Comment by bluebill1049** [comment]
can you provide a csb?

**Reply by mwlai288** [reply]
sorry, what is a csb?

**Reply by bluebill1049** [reply]
oops, i mean codesandbox.

**Reply by mwlai288** [reply]
I got it working.

**Comment by mwlai288** [comment, marked as answer]
Fixed

**Reply by bluebill1049** [reply]
would like to share your solution?

**Reply by mwlai288** [reply]
I actually had to update my formErrors file in my work to fix the error.

**Reply by bluebill1049** [reply]
kk
